prow/config.yaml

```diff
@@ -41,6 +41,21 @@ branch-protection:
 
 tide:
   queries:
+  - repos:
+    - kubeflow/community
```

Comment:
This repo does not have a root level OWNERS file yet which will be needed in order for the approval process to work.

Reply:
Thanks. We'll add one. Any reason I need to do that before this PR?

Reply:
Shouldn't matter much either way, but nothing will be able to get the approved label in that repo until the there is an OWNERS file so you'd have to click the merge button to merge the PR to add the OWNERS file.

Reply:
Ack